core/table_configs/pathfinders/round_1.py

```diff
@@ -803,9 +777,7 @@ class PFErrors:
                 "Actual, forecast or cancelled": pa.Column(
                     str, pa.Check.isin(PFEnums.ACTUAL_FORECAST, error=PFErrors.ISIN)
                 ),
-                "Reporting period change takes place": pa.Column(
-                    str, pa.Check.isin(PFEnums.REPORTING_PERIOD, error=PFErrors.ISIN)
-                ),
+                "Reporting period change takes place": pa.Column(str),
```

Comment:
The enum check has been removed but not replaced by any check in cross-table validation - although a reporting period check has been added there, it only checks the reporting period in the "Reporting period" table of the "Start" worksheet, not this one